### ISSUE TITLE
Accessibility missing language

### ISSUE THREAD

**Issue by FilippeMafra** [FilippeMafra/css-example#4]
<img width="959" alt="Image" src="https://github.com/user-attachments/assets/9b40000f-53d6-4f6c-918c-28412815d52b" />

**Comment by FilippeMafra** [comment]
https://github.com/FilippeMafra/css-example/pull/6

<img width="506" alt="Image" src="https://github.com/user-attachments/assets/ee22b845-7fd9-4f72-a8c3-37f86e7dddc8" />
